### DISCUSSION TITLE
License not found for target

### DISCUSSION THREAD

**Discussion by yiziai** [maplibre/maplibre-native#913] [Q&A]
as common action ,download maplibre with cmd git clone --recurse-submodules -j8 https://github.com/maplibre/maplibre-gl-native.git,
then open platform/android with andriod studio,  push excute button,   error happend :
CMake Error at C:/workspace/gisData/test/maplibre-gl-native/scripts/license.cmake:24 (message):
License not found for target: Mapbox::Base::geojson-vt-cpp
Call Stack (most recent call first):
C:/workspace/gisData/test/maplibre-gl-native/scripts/license.cmake:40 (mbgl_generate_license)
C:/workspace/gisData/test/maplibre-gl-native/CMakeLists.txt:1089 (include)

how to slove it, thanks any tip

**Comment by yiziai** [comment]
download submodule fail

**Reply by louwers** [reply]
Did you solve it?

If not can you post the entire stack trace?
